code/modules/mod/modules/modules_visor.dm

```diff
@@ -91,10 +91,10 @@
 	incompatible_modules = list(/obj/item/mod/module/night, /obj/item/mod/module/visor/night)
 	required_slots = list(ITEM_SLOT_HEAD|ITEM_SLOT_EYES|ITEM_SLOT_MASK)
 
-/obj/item/mod/module/night/on_activation()
-	ADD_TRAIT(mod.wearer, TRAIT_TRUE_NIGHT_VISION, REF(src))
+/obj/item/mod/module/night/on_activation(mob/activator)
+	mod.wearer.add_traits(mod.wearer, TRAIT_TRUE_NIGHT_VISION, REF(src))
```

Comment:
why this change?

Reply:
It's from this PR.
https://github.com/tgstation/tgstation/pull/92506